bin/release.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
+
+function usage {
+    echo
+    echo "Builds all the images for the specified kimai tag and then tags them and pushes"
+    echo "them:"
+    echo "  fpm-alpine-1.10.2-prod, latest, latest-prod"
+    echo "  fpm-alpine-1.10.2-dev"
+    echo "  apache-debian-1.10.2-prod"
+    echo "  apache-debian-1.10.2-dev, latest-dev"
+}
+
+USAGE="$0 TAG"
+TAG=$1
+
+if [ -z "$TAG" ]; then
+  echo $USAGE
+  exit 1
+fi
+
+if [ "$TAG" == "-h" ] || [ "$TAG" == "help" ] || [ "$TAG" == "--help" ]; then # <-- maybe switch to getopts
+  echo $USAGE
+  usage
+  exit 0
+fi
+
+BIN_DIR=$(dirname $0)
+
+$BIN_DIR/build.sh -c $TAG
```

Comment:
Haven't had a look in these scripts, but what about a mini check if the given TAG looks like a valid version number before proceeding?

Reply:
Yes, as this may fail if there is a space in `${TAG}` for the line `docker tag ...${TAG}...`.

May be some validation using `RegEx` could do the trick.